tests/test_docker.py

```diff
@@ -88,7 +88,45 @@ def test_instance_activate_need_pull_image(agent, responses):
 def test_instance_only_activate(agent, responses):
     _delete_container('/c861f990-4472-4fa1-960f-65171b544c28')
 
+    def pre(req):
+        instance = req['data']['instanceHostMap']['instance']
+        for nic in instance['nics']:
+            nic['macAddress'] = ''
+
+    def post(req, resp):
+        docker_inspect = resp['data']['instance']['+data']['dockerInspect']
+        mac_received = docker_inspect['Config']['MacAddress']
+        mac_nic_received = docker_inspect['NetworkSettings']['MacAddress']
+        assert mac_received == ''
+        assert mac_nic_received is not None
+        del resp['data']['instance']['+data']['dockerInspect']
+        docker_container = resp['data']['instance']['+data']['dockerContainer']
+        fields = resp['data']['instance']['+data']['+fields']
+        del docker_container['Created']
```

Comment:
We have a common function for deleting these common fields:
```container_field_test_boiler_plate```
use that instead of doing it yourself